app/api/helpers/calendar/ical.py

```diff
@@ -39,10 +37,8 @@ def to_ical(event, include_sessions=False, my_schedule=False):
             .order_by(Session.starts_at.asc())
         )
         if my_schedule:
-            if not current_user:
```

Comment:
It should also work with current user

Reply:
So we are also keeping the jwt_optional header then too right? :sweat_smile:

Reply:
Yes